src/mcs/mcs_types.jl

```diff
@@ -231,6 +231,5 @@ end
 
 function Base.getindex(sim_inst::SimulationInstance, comp_name::Symbol, datum_name::Symbol; model::Int = 1)
     msg = "getindex method for `SimulationInstance` has been replaced with getdataframe function for consistency of return type, please use getdataframe instead"
-    Base.depwarn("$msg, $(stacktrace())", :getindex)
-    return sim_inst.results[model][(comp_name, datum_name)]
+    error("$msg")
```

Comment:
```suggestion
    error(msg)
```
:) Or don't use a local var at all and just pass the string directly to `error`.

Reply:
All relics of older more complex messages from depreciations :) ... I’ll go through these today